### ISSUE TITLE
Make utility mode non-interactive

### ISSUE THREAD

**Issue by gaul** [s3fs-fuse/s3fs-fuse#945]
Utility mode allows listing and aborting incomplete MPU via user input:

```
$ s3fs mybucket -u
Lists the parts that have been uploaded for a specific multipart upload.

---------------------------------------------------------------
 Path     : /128gb
 UploadId : JzQEgxwURH8h9XU9Cpa9uNFpTFhjICW7e5cBW7RmmgW9v_1TlVGqlU2F27g6r21DW9Lw97JzsPxN.bsh.pwN3fmyQNnZlQ_u006LD5ds0uQ-
 Date     : 2019-01-31T04:25:44.000Z

---------------------------------------------------------------
Would you remove all objects? [Y/N]
```

This only allows interactive uses and thus cannot be part of our CI tests.  Instead could s3fs break the operations into two, e.g., `--list-incomplete-mpu` and `--abort-incomplete-mpu`, perhaps with the latter taking a `older-than-day` parameter?  Alternatively, s3fs could automatically abort incomplete MPU older than some number of days like goofys does.

**Comment by ggtakec** [comment]
@gaul Thanks for your suggestion.

I think that adding "--list-incomplete-mpu" option is good.
About the "--abort-incomplete-mpu" option, I think this option should have parameters of "XX[YMDhm]" or "all" as its arguments.

The "-u" option can not be deleted because it is backward compatible.
But what about assigning "--incomplete-mpu" as an equivalent option?
Along with that, all option name prefix has "--incomplete-mpu", how about as follows?
```
--incomplete-mpu(-u)
--incomplete-mpu-list
--incomplete-mpu-abort ([1-9][0-9][Y|M|D|h|m] | all)
```
_"YMDhm" is the year/month/day/hour, and "all" is all._

**Comment by ggtakec** [comment]
@gaul This issue is fixed by #951 , then this will be closed.
If you have more problem about this, please reopen.
